doc/source/development/contributing.rst

```diff
@@ -243,8 +243,16 @@ You'll need to have at least python3.5 installed on your system.
    # Create a virtual environment
    # Use an ENV_DIR of your choice. We'll use ~/virtualenvs/pandas-dev
    # Any parent directories should already exist
+   # If you are using Windows and command prompt, replace ~ (the tilde sign)
+   # with %userprofile%
    python3 -m venv ~/virtualenvs/pandas-dev
+
    # Activate the virtualenv
+   # If you are using Windows, you can find the activation scripts under
+   #     ~\virtualenvs\pandas-dev\scripts
+   # Please refer to the official user guide at
+   # https://virtualenv.pypa.io/en/stable/userguide/#activate-script
+   # about how to activate your virtual environment under Windows
```

Comment:
Thanks for clarifying this @gbaychev.

Don't you think could be clearer to have the windows instructions in a separate block? Feels like there is too much information in this block now. And having two blocks would let users just read the one they care about, and have the information presented in a clearer way. 

Feel free to disagree, but I think we can probably do better at communicating this.

Reply:
You guys are very careful and diplomatic, cheers for that :) Yes @datapythonista,  you are absolutely correct, I've split the information into two code-blocks, please see 39f6fa7